spec/support/capybara_selenium.rb

```diff
@@ -1,21 +1,26 @@
 # frozen_string_literal: true
-require "capybara/rspec"
+
+require 'capybara/rails'
+require 'capybara/rspec'
 require "selenium-webdriver"
 
 Capybara.register_driver :chrome do |app|
   Capybara::Selenium::Driver.new(app, browser: :chrome)
 end
+
 Capybara.register_driver(:selenium) do |app|
   capabilities = Selenium::WebDriver::Remote::Capabilities.chrome(
     chromeOptions: { args: %w[headless disable-gpu disable-setuid-sandbox window-size=7680,4320] }
   )
+
   browser_options = ::Selenium::WebDriver::Chrome::Options.new
   browser_options.args << "--headless"
```

Comment:
This might not be necessary, I couldn't manage to get this to run headlessly for my local dev. environment (nor in CircleCI).